### PR TITLE
Not as much as I thought, since two warnings are legit, only the inco…

### PR DESCRIPTION
**Pull request by stupleb**
…rrectly commented one can be fixed

### CODE REVIEW

File: site/source/service-guides/lightning/fully-noded-lnd.rst
Comment (Dominion5254) [inline site/source/service-guides/lightning/fully-noded-lnd.rst:18]: I'm not sure I understand this PR. These two lines seem like pertinent information we want included in the guide rather than comments with the incorrect syntax.

That said, line 16 should maybe just be a single `#` character to include it in the numbered list. Then `[Pasting Credentials]` might appear as a either a separate note or section with the subsequent lines about the 4 fields to be copy/pasted.

Comment (stupleb) [reply site/source/service-guides/lightning/fully-noded-lnd.rst:18]: Right... I was absolutely convinced that these were commented out do to a change and that copying and pasting was the recommended way. It's been malformed for some time (https://docs.start9.com/0.3.5.x/service-guides/lightning/fully-noded-lnd#fully-noded-lnd)

Perhaps the were meant to be tabs at one point? You know, I think that make sense.

Comment (stupleb) [reply site/source/service-guides/lightning/fully-noded-lnd.rst:18]: @Dominion5254 OK check it now, I'm pretty sure these were tabs that were never converted to reStructuredText format.

**Review by Dominion5254 — DISMISSED** [review]
The tabs look great 👍

**Comment by Dominion5254** [comment]
Actually, it looks like there is one merge conflict to be resolved before this PR can be merged.

It should be straightforward to fix, but let me know if you have questions.

**Review by stupleb — COMMENTED** [review]
Resolved merge conflict

**Comment by stupleb** [comment]
@Dominion5254 needs yet another review

**Review by Dominion5254 — APPROVED** [review]

**Pull request merged**
